controller/src/beerocks/master/db/db.cpp

```diff
@@ -2833,6 +2833,13 @@ bool db::clear_client_persistent_db(const sMacAddr &mac)
     return true;
 }
 
+bool db::is_hostap_on_selected_bands(beerocks::eFreqType selected_bands, const sMacAddr &hostap)
+{
+    //todo need to change selected_bands type to eClientSelectedBands when merged to master
+    return (selected_bands ==
+            wireless_utils::which_freq(get_node_channel(tlvf::mac_to_string(hostap))));
+}
```

Comment:
```suggestion
    //todo need to change selected_bands type to eClientSelectedBands when merged to master
    if (selected_bands == DISABLED || stay_on_seleceted_bands != eTriStateBool:Enabled) {
        return true;
    }
    return (selected_bands ==
            wireless_utils::which_freq(get_node_channel(tlvf::mac_to_string(hostap))));
}
```

Reply:
is `stay_on_seleceted_bands` global, or do we need to pass it along as an argument

Reply:
aligned to `uint8_t selected_bands for now.
need to update eFreqType and use eClientSelectedBands format instead
https://jira.prplfoundation.org/browse/PPM-353